### PR TITLE
[FORUM-1251] Merge wars of Forum Project

### PR DESCRIPTION
**Pull request by mtrabelsi**

### CODE REVIEW

**Comment by thomasdelhomenie** [comment]
Moving Poll portlet in forum/webapp changes the portlet reference, so it breaks backward compatibility (same problem as the one in ECMS). We cannot do that until we find a solution to keep the backward compatibility.

File: extension/pom.xml [hunk 1]
```diff
@@ -12,6 +12,5 @@
   <description>KS Extension Root Pom</description>
   <modules>
     <module>config</module>
```
Comment (thomasdelhomenie) [inline extension/pom.xml]: config module (an actually all the extension folder) can also be removed.

File: forum/webapp/src/main/resources/tomcat/forum.xml [hunk 2]
```diff
@@ -1,2 +1,4 @@
 <Context path="/forum" docBase="../../../eXoProjects/ks/trunk/eXoApplication/forum/webapp/src/main/webapp" 
-         debug="0" reloadable="true"/>
\ No newline at end of file
+         debug="0" reloadable="true"/>
+<Context path="/forum" docBase="../../../eXoProjects/ks/trunk/web/forum/src/main/webapp" 
+         debug="0" reloadable="true" />
```
Comment (thomasdelhomenie) [inline forum/webapp/src/main/resources/tomcat/forum.xml]: Is this really necessary ? The path docBase does not even exist. I think this file can be deleted.

File: poll/pom.xml [hunk 3]
```diff
@@ -11,6 +11,5 @@
   <name>Forum Poll</name>
   <modules>
     <module>service</module>
```
Comment (thomasdelhomenie) [inline poll/pom.xml]: We should also move poll services in forum/service IMO.

Comment (mtrabelsi) [inline poll/pom.xml]: Thanks for pointing out about this, but this is a **jar** so normally we should treat it in jar merging cycle.?

Comment (thomasdelhomenie) [inline poll/pom.xml]: It is a consequence of the wars merge. Leaving this jar here would be quite inconsistent, so I think we should handle it in the same task.

File: extension/pom.xml (hunk 1 above)
Comment (mtrabelsi) [inline extension/pom.xml]: done.

File: forum/webapp/src/main/resources/tomcat/forum.xml (hunk 2 above)
Comment (mtrabelsi) [inline forum/webapp/src/main/resources/tomcat/forum.xml]: done.

**Pull request merged**